src/dune_rules/melange_stanzas.ml

```diff
@@ -8,7 +8,7 @@ module Emit = struct
     ; alias : Alias.Name.t option
     ; module_system : Melange.Module_system.t
     ; entries : Ordered_set_lang.t
-    ; libraries : Lib_dep.t list
+    ; libraries : Loc.t * Lib_dep.t list
```

Comment:
I don't think this location is needed anymore.

Reply:
fixed in https://github.com/ocaml/dune/pull/6412/commits/80ce1ee4f968a8ecb457b13cf68b50b99b44ef56